lib/random.gd

```diff
@@ -69,9 +69,84 @@ DeclareCategory( "IsRandomSource", IsComponentObjectRep );
 ##  </ManSection>
 ##  <#/GAPDoc>
 ##
-DeclareOperation( "Random", [IsRandomSource, IsList] );
+DeclareOperation( "Random", [IsRandomSource, IsListOrCollection] );
```

Comment:
It is covered. This just lines up two-argument random with one argument random, which was already `DeclareOperation( "Random", [ IsListOrCollection ] );`. It's needed to take random elements from lists which aren't collection.